shared/jni/setupBuild.gradle

```diff
@@ -64,6 +64,12 @@ model {
                 }
                 it.cppCompiler.define 'WPILIB_EXPORTS'
                 it.cCompiler.define 'WPILIB_EXPORTS'
+                if (nativeName != 'wpiutil') {
+                    it.cppCompiler.define 'PROTOBUF_USE_DLLS'
```

Comment:
Is this going to break not having this defined in robot projects or vendor deps?

Reply:
It shouldn't. The reason I'm doing IMPORTS here is to avoid the compiler complaining about the derived class being marked as exported when the base class is not marked as exported.